lib/Configuration.php

```diff
@@ -1,180 +1,153 @@
 <?php
-/**
- * Configuration
- * PHP version 5
- *
- * @category Class
- * @package  Upcloud\ApiClient
- */
-
-/**
- * Upcloud api
- *
- * The UpCloud API consists of operations used to control resources on UpCloud. The API is a web service interface. HTTPS is used to connect to the API. The API follows the principles of a RESTful web service wherever possible. The base URL for all API operations is  https://api.upcloud.com/. All API operations require authentication.
- *
- * OpenAPI spec version: 1.2.0
- * 
- */
 
+declare(strict_types=1);
 
 namespace Upcloud\ApiClient;
 
+use InvalidArgumentException;
+
 /**
  * Configuration Class Doc Comment
- * PHP version 5
  *
  * @category Class
- * @package  Upcloud\ApiClient
  */
 class Configuration
 {
     private static $defaultConfiguration;
 
     /**
-     * Associate array to store API key(s)
+     * Associate array to store API key(s).
      *
      * @var string[]
      */
     protected $apiKeys = [];
 
     /**
-     * Associate array to store API prefix (e.g. Bearer)
+     * Associate array to store API prefix (e.g. Bearer).
      *
      * @var string[]
      */
     protected $apiKeyPrefixes = [];
 
     /**
-     * Access token for OAuth
+     * Access token for OAuth.
      *
      * @var string
      */
     protected $accessToken = '';
 
     /**
-     * Username for HTTP basic authentication
+     * Username for HTTP basic authentication.
      *
      * @var string
      */
     protected $username = '';
 
     /**
-     * Password for HTTP basic authentication
+     * Password for HTTP basic authentication.
      *
      * @var string
      */
     protected $password = '';
 
     /**
-     * The host
+     * The host.
      *
      * @var string
      */
-    protected $host = 'https://api.upcloud.com/1.2';
+    protected $host = 'https://api.upcloud.com/1.3';
 
     /**
-     * User agent of the HTTP request, set to "PHP-Swagger" by default
+     * User agent of the HTTP request
      *
      * @var string
      */
-    protected $userAgent = 'Swagger-Codegen/1.0.0/php';
+    protected $userAgent = 'Upcloud-Sdk/1.3/php';
```

Comment:
We have a new format for user agents `$reponame/$releaseversion` so in this case `upcloud-php-api/v1.0.3` (v1.0.3 is the latest, but this PR will bump up major versioning)